docs/modules/installation/pages/requirements.adoc

```diff
@@ -1,18 +1,17 @@
 
 = Requirements
 
-== Runtime requirements
+This section describes the requirements for installing {page-component-title}.
 
-* OpenNMS {page-component-title} works with http://docs.grafana.org/installation[Grafana Server] version {grafana-version-required} and up.
-It has been tested with Grafana versions up to {grafana-version-tested} at the time of release.
+== Runtime requirements
 
-The Grafana Dashboard Report feature in Meridian/Horizon requires the
-https://grafana.com/grafana/plugins/grafana-image-renderer/[Grafana Image Renderer plugin] to be installed.
+For runtime operations, you need to install and set up either https://www.opennms.org[OpenNMS Horizon] or https://www.opennms.com[Meridian].
```

Comment:
@aramosvizcarra, could you link the version compatibility matrix you mentioned earlier?

Reply:
I don think we have something official, just but since the OpenNMS docs only document supported version I would also do the same for the version we use. Somthing like this
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/ARamos-Vizcarra/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/ARamos-Vizcarra/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>
<!--table
	{mso-displayed-decimal-separator:"\.";
	mso-displayed-thousand-separator:"\,";}
@page
	{margin:.75in .7in .75in .7in;
	mso-header-margin:.3in;
	mso-footer-margin:.3in;}
tr
	{mso-height-source:auto;}
col
	{mso-width-source:auto;}
br
	{mso-data-placement:same-cell;}
td
	{padding-top:1px;
	padding-right:1px;
	padding-left:1px;
	mso-ignore:padding;
	color:black;
	font-size:12.0pt;
	font-weight:400;
	font-style:normal;
	text-decoration:none;
	font-family:Calibri, sans-serif;
	mso-font-charset:0;
	mso-number-format:General;
	text-align:general;
	vertical-align:bottom;
	border:none;
	mso-background-source:auto;
	mso-pattern:auto;
	mso-protection:locked visible;
	white-space:nowrap;
	mso-rotate:0;}
-->
</style>
</head>

<body link="#0563C1" vlink="#954F72">



Horizon | Meridian | Helm | Grafana
-- | -- | -- | --
31 | 2023 | 9.x+ | 9.x+
31 | 2023 | 8.x | 8.x
30 | 2022 | 8.x | 8.x
29 | 2021 | 7.x | 7.x



</body>

</html>